hq/app/model/Serializers.scala

```diff
@@ -7,7 +7,19 @@ object Serializers {
 
   implicit val snykErrorFormat: Reads[SnykError] = Json.reads[SnykError]
 
-  implicit val snykOrgFormat: Reads[SnykOrganisation] = Json.reads[SnykOrganisation]
+  implicit val snykGroupFormat: Reads[SnykGroup] = (
```

Comment:
I think these can be automatically derived for you. (Less code is better code!)

```scala
implicit val snykGroupFormat: Reads[SnykGroup] = Json.reads[SnykGroup]
implicit val snykOrgFormat: Reads[SnykOrganisation] = Json.reads[SnykOrganisation]
```

This is true iff the case class's field names exactly match the JSON field names.

Reply:
SnykGroup can, SnykOrganisation can't.  I have tests to prove it :)

Done.